config/bref.php

```diff
@@ -30,4 +30,17 @@
 
     'request_context' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Use AWS credential tokens
+    |--------------------------------------------------------------------------
+    |
+    | Laravel does not use AWS_SESSION_TOKEN environment vars by default
+    | Bref can automatically add these tokens into your configuration
+    | If you are not using AWS Roles you may need to disable this.
```

Comment:
Uploading AWS credentials in Lambda is a very last resort option honestly, I'd love to know when it's not possible to use the Lambda credentials?

Anyway, I'd optimize for being super explicit that most users should not do this/use this, unless they definitely know why they are doing this. So I'd reword probably like this:

```suggestion
    | Bref fixes Laravel to use the AWS_SESSION_TOKEN environment variable
    | provided by AWS Lambda automatically (required for AWS credentials to work).
    | You should disable that only if you set your own AWS access keys manually
    | (which is not recommended in most cases).
```

Reply:
Totally agree it's a last resort. Our AWS setup has a lot of legacy and we have many clients with DKIM etc in SES in one region/account, but for various reasons we're also sending emails from another AWS account. We use roles for most things, but in this case we need to use different credentials for connecting to SES.